hr_timesheet_employee_cost_contract/tests/test_hr_timesheet_employee_cost_contract.py

```diff
@@ -104,24 +104,53 @@ def test_annual_avg(self):
                     'name': 'Employee Contract #1',
                     'wage': 3000.0,
                     'state': 'open',
-                    'date_start': date(2020, 1, 1),
+                    'date_start': date(self.today.year, 1, 1),
                 }),
                 (0, 0, {
                     'name': 'Employee Contract #2',
                     'wage': 1000.0,
                     'state': 'open',
-                    'date_start': date(2020, 1, 1),
+                    'date_start': date(self.today.year, 1, 1),
                 }),
             ],
         })
-        as_of_day = date(2020, 12, 1)
+        as_of_day = date(self.today.year, 12, 1)
```

Comment:
@hbrunn here can be a mistake of mine, 31 instead of 1

Reply:
hmm, doesn't make a difference. I added https://github.com/OCA/timesheet/pull/432/commits/6b1d443d9e805334dbfb2b66a0a3d3100a4f2c5d to actually test the calendar year, but that doesn't change the averages